accounts/abi/bind/base.go

```diff
@@ -30,6 +30,10 @@ import (
 	"github.com/XinFinOrg/XDPoSChain/event"
 )
 
+var (
+	errNoEventSignature = errors.New("no event signature")
```

Comment:
why define an error here instead of return fmt.Errorf("..") like the rest of the file?

Reply:
I'm following the code from eth. https://github.com/ethereum/go-ethereum/blob/master/accounts/abi/bind/base.go#L38